python/spacewalk/satellite_tools/repo_plugins/yum_src.py

```diff
@@ -31,6 +31,7 @@
 import lzma
 import os
 import re
+import requests
```

Comment:
Is it needed to include `requests`? I'm asking because we already use `urlgrabber` in reposync and, while I'd like to get rid of it, I think it's better to stick to one http library if possible.

Reply:
Yeah, I was hoping to avoid importing anything but I couldn't find a tidy way to get the content-type headers with urlgrabber. It seemed like it was possible to _supply_ headers but not to retrieve them in a consistent format; I think you'd have to parse the page itself looking for something starting <!DOCTYPE> or <?xml> and then map it to types by hand, and that seemed fragile... So unless I missed something with urlgrabber I think requests is the less messy option.